src/geometry/LinearPropagator.i.hh

```diff
@@ -28,12 +28,16 @@ CELER_FUNCTION LinearPropagator::LinearPropagator(GeoTrackView& track)
  * Move track by next_step(), which takes it to next volume boundary.
  */
 CELER_FUNCTION
-void LinearPropagator::operator()()
+LinearPropagator::OutputType LinearPropagator::operator()()
 {
-    this->apply_linear_step(track_.next_step() + track_.tolerance());
+    OutputType result;
+    result.distance =
+      this->apply_linear_step(track_.next_step() + track_.tolerance());
```

Comment:
This file needs to be formatted correctly (git-clang-format and the git hooks should help you with this).

Reply:
Thanks for pointing that out. I had not ran the commit-hook script in this area.  However there was no change into this particular line!